MyLab6.playground/Contents.swift

```diff
@@ -0,0 +1,44 @@
+
+var toDoList = [1,10,22,44,5,13,19,23,99,9]
+ toDoList.reverse()
+ print(toDoList)
+ toDoList.randomElement()
```

Comment:
great efforts @Fatan1996 , but what does the .randomElement() do for us? we needed to have the array to be ordered in a random order is that happening? (hint search for a method that do that for array)